packages/components/src/dropdown-menu-v2-ariakit/index.tsx

```diff
@@ -58,13 +57,19 @@ export const DropdownMenuItem = forwardRef<
 			hideOnClick={ hideOnClick }
 			store={ dropdownMenuContext?.store }
 		>
-			{ prefix && (
-				<Styled.ItemPrefixWrapper>{ prefix }</Styled.ItemPrefixWrapper>
-			) }
-			{ children }
-			{ suffix && (
-				<Styled.ItemSuffixWrapper>{ suffix }</Styled.ItemSuffixWrapper>
-			) }
+			<Styled.ItemPrefixWrapper>{ prefix }</Styled.ItemPrefixWrapper>
```

Comment:
The prefix wrapper is always rendered (even when empty) in order to correctly render in the first column of the grid layout.